x-pack/plugins/security/public/components/management/users/confirm_delete.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import { EuiOverlayMask, EuiConfirmModal } from '@elastic/eui';
+import { toastNotifications } from 'ui/notify';
+export class ConfirmDelete extends Component {
+  deleteUsers = () => {
+    const { usersToDelete, apiClient, callback } = this.props;
+    const errors = [];
+    usersToDelete.forEach(async username => {
+      try {
+        await apiClient.deleteUser(username);
+        toastNotifications.addSuccess(`Deleted user ${username}`);
+      } catch (e) {
+        errors.push(username);
+        toastNotifications.addDanger(`Error deleting user ${username}`);
+      }
+      if (callback) {
+        callback(usersToDelete, errors);
+      }
+    });
+  };
+  render() {
+    const { usersToDelete, onCancel } = this.props;
+    const moreThanOne = usersToDelete.length > 1;
+    const title = moreThanOne
+      ? `Delete ${usersToDelete.length} users`
+      : `Delete user '${usersToDelete[0]}'`;
+    return (
+      <EuiOverlayMask>
+        <EuiConfirmModal
```

Comment:
@cchaos This modal looks like it's missing `flex-direction: column`. I reinstalled the dependencies. Is anyone else seeing this?

![image](https://user-images.githubusercontent.com/1238659/42897456-ed087218-8a74-11e8-9ae6-c89f6370c5d1.png)

Reply:
I guess this will be fixed by the followup PR updating EUI in X-Pack.